assets/src/blocks/Gallery/GalleryCarousel.js

```diff
@@ -125,9 +125,8 @@ export const GalleryCarousel = ({ images, onImageClick, isEditing }) => {
               style={{ objectPosition: image.focus_image }}
               alt={image.alt_text}
               title={image.alt_text}
-              onClick={() => {
```

Comment:
This is basically a performance tuning, since this lambda function is created every time the `render` method is invoked for the only reason to send the `index` value.